packages/react/README.md

```diff
@@ -15,3 +15,226 @@
 # @solana/react
 
 This package contains React hooks for building Solana apps.
+
+## Hooks
```

Comment:
I'm really tired; can you help me proofread this?

Reply:
Generally looks fine to me at a first pass!